### DISCUSSION TITLE
How to do `composer install` with `medialibrary-pro` in Github Actions

### DISCUSSION THREAD

**Discussion by maxonfjvipon** [spatie/laravel-medialibrary#3177] [Q&A]
Hello there!
I use laravel-medialibrary-pro. It can be downloaded if you has a special authorization key.
If I try to publish it (with git push) key will be changed. 

So I want to run tests on my CI/CD pipeline using Github Actions. Before I do `./vendor/bin/phpunit` I need to do `composer install`, but it fails because there's no auth key to download it

Is there a workaround?

**Comment by WhereIsLucas** [comment]
Hi,
Did you find a solution for this ?
Thanks !
